src/Formatter/AbstractFormatter.php

```diff
@@ -48,33 +34,4 @@ public static function accepts()
      * @return string
      */
     abstract protected function body(PayloadInterface $payload);
-
-    /**
-     * Get the response status from the payload.
-     *
-     * @param PayloadInterface $payload
-     *
-     * @return integer
-     */
-    public function status(PayloadInterface $payload)
```

Comment:
I thought about making this into an interface now that there are no implemented methods, but that's not possible with protected methods.